### PR TITLE
Replacing latexdefs

### PR DESCRIPTION
**Pull request by GinoGiotto**
I made some progress towards #3067, this pull request doesn't address all issues. I'll write a comment about what is left and what I think is already done.

### CODE REVIEW

**Comment by GinoGiotto** [comment]
I noticed that latex editors don't add spacing for adjacent latexdefs to <code>\text{...}</code> . I propose to add some kind of spacing inside, like <code>\text{\\,text\\,}</code> or simply <code>\text{ text }</code>.

File: iset.mm [hunk 1]
```diff
@@ -129413,7 +129413,7 @@ Norman Megill (2007) section 1.1.3.  Megill then states, "A number of
   latexdef "F/_" as "\underline{\Finv}";
 htmldef "CondEq" as "CondEq";
   althtmldef "CondEq" as "CondEq";
-  latexdef "CondEq" as "\mbox{CondEq}";
+  latexdef "CondEq" as "\text{CondEq}";
```
Comment (benjub) [inline iset.mm:129416]:
```suggestion
  latexdef "CondEq" as "\operatorname{CondEq}";
```

File: iset.mm [hunk 2]
```diff
@@ -129560,7 +129560,7 @@ Norman Megill (2007) section 1.1.3.  Megill then states, "A number of
   althtmldef ".0b" as
     ' <SPAN CLASS=symvar STYLE="border-bottom:1px dotted;color:#C3C">' +
     '&#x1D7CE</SPAN> ';
-  latexdef ".0b" as "\mbox{\boldmath$0$}";
+  latexdef ".0b" as "\text{\boldmath$0$}";
```
Comment (benjub) [inline iset.mm:129563]:
```suggestion
  latexdef ".0b" as "\pmb{0}";
```

File: nf.mm [hunk 3]
```diff
@@ -63228,32 +63228,32 @@ the first case of his notation (simple exponentiation) and subscript it
...
 htmldef "2nd" as
     "<IMG SRC='_2nd.gif' WIDTH=21 HEIGHT=19 TITLE='2nd' ALIGN=TOP>";
   althtmldef "2nd" as '2<SUP>nd</SUP> ';
-  latexdef "2nd" as "2^{\rm nd}";
+  latexdef "2nd" as "2^\mathrm{nd}";
 htmldef "Swap" as
     "<FONT FACE=sans-serif> Swap </FONT>";
   althtmldef "Swap" as '<FONT FACE=sans-serif> Swap </FONT>';
-  latexdef "Swap" as "{\rm Swap}";
+  latexdef "Swap" as "\mathrm{Swap}";
```
Comment (benjub) [inline nf.mm:63248]: can you remove the blank lines?

**Comment by benjub** [comment]
I made a few comments. I saw some other "clumsy" latexdefs (among the unchanged ones), but work has to be done twice: in both set.mm and iset.mm...
This will be a large task, and tex ouputting is not used often (at least, not by me), so I'm not sure it's worth doing too much, especially as long as we do not have a way to share latexdefs between multiple databases.

**Comment by GinoGiotto** [comment]
@benjub btw now I'm also unable to use the "commit suggestion" button, it gives me the same error you mentioned. I don't know why.

**Comment by GinoGiotto** [comment]
> I noticed that latex editors don't add spacing for adjacent latexdefs to `\text{...}` . I propose to add some kind of spacing inside, like `\text{\,text\,}` or simply `\text{ text }`.

@benjub what's your opinion about this? What spacing would you suggest?

**Comment by benjub** [comment]
> I noticed that latex editors don't add spacing for adjacent latexdefs to \text{...} . I propose to add some kind of spacing inside, like \text{\,text\,} or simply \text{ text }.
> @benjub what's your opinion about this? What spacing would you suggest?

Between both, the latter, although probably in most cases, something else is meant, e.g., `\mathbin{text}` or `\mathrel{text}`.

This PR looks good to me.

**Comment by GinoGiotto** [comment]
> Between both, the latter, although probably in most cases, something else is meant, e.g., `\mathbin{text}` or `\mathrel{text}`.

@benjub I'm changing <code>\text{text}</code> into <code>\text{ text }</code>, until further analysis I think spacing is needed because those latexdefs together appear like a single word.

**Comment by GinoGiotto** [comment]
Ok, unless there are further commit suggestions, I've finished here

**Review by benjub — APPROVED** [review]
LGTM as an intermediate step.

**Comment by benjub** [comment]
If no one objects (or merges by then), I'll merge tonight UTC. Thanks @GinoGiotto for all this work.

**Pull request merged**